src/components/Banner/Banner.scss

```diff
@@ -10,20 +10,25 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tight);
 @mixin banner-attributes($highlight, $background, $in-page, $tint) {
   transition: box-shadow duration() easing();
   transition-delay: duration(fast);
-  border-radius: var(--p-border-radius-wide, inherit);
   box-shadow: var(--p-banner-border, none);
 
   @media (-ms-high-contrast: active) {
     outline: 1px solid;
   }
 
   @if $in-page {
+    border-radius: var(
+      --p-border-radius-wide,
+      0 0 border-radius() border-radius()
```

Comment:
I tophatted the banners in story book but didn't see a version where the top left and right didn't have a radius. Do we need an example for this as well?

Reply:
This PR is to fix current banners. All the default (in-page) banners have square corners on top. In-context or in-card banners have all rounded corners.

<img width="503" alt="Screen Shot 2020-05-11 at 11 07 22 AM" src="https://user-images.githubusercontent.com/344839/81595615-9d705f00-9377-11ea-8257-59e7f7a92d13.png">